iOS/Flows/Room/RoomCollectionViewDataSource.swift

```diff
@@ -54,6 +81,24 @@ class RoomCollectionViewDataSource: CollectionViewDataSource<RoomSectionType, Ro
                                            indexPath: IndexPath) -> UICollectionReusableView? {
         
         switch section {
+        case .members:
+            if kind == UICollectionView.elementKindSectionHeader {
+                let header = collectionView.dequeueConfiguredReusableSupplementary(using: self.memberHeaderConfig, for: indexPath)
+                header.leftLabel.setText("Favorites")
+                header.button.didSelect { [unowned self] in
+                    self.didSelectAddPerson?()
+                }
+                return header
+            } else if kind == UICollectionView.elementKindSectionFooter {
```

Comment:
Added a footer and header to the members section as the conversations already has a  segment header, and a background that would be extended. This give the illusion of header for the conversations.